src/pages/case-studies/codecov-uses-sourcegraph-to-resolve-incidents-12-times-faster.tsx

```diff
@@ -0,0 +1,263 @@
+                    Sourcegraph helps developers help themselves and each other. “I want developers to only have to help
+                    each other when necessary,” said Jerrod. Sourcegraph searches have helped Codecov developers answer
+                    questions ahead of time, before they have to ask for help and wait, often across time zones, for a
+                    response.
+                </p>
+                <p>
+                    Efficiency is only half the benefit. According to Jerrod, “We increased the interpersonal trust and
+                    we increased the interpersonal dynamics in the company, making people feel more welcomed.”
+                </p>
+                <p>
+                    Asking for help, of course, isn't a practice exclusive to early career developers. “Sourcegraph will
+                    not only help with their onboarding,” said Mitchell. “It'll help post-onboarding, increasing the
+                    developers' usability across the repositories we have.”
+                </p>
+            </ContentSection>
+
+            <div className="bg-gradient-saturn py-7">
+                <ContentSection>
+                    <ThreeUpText title="Results" items={threeUpTextItems} />
+                </ContentSection>
+            </div>
+
+            <ContentSection color="white" className="py-7 max-w-700 col-xl-5">
+                <h3 className="mb-4 display-4 font-weight-normal max-w-650">
+                    Sourcegraph was the right fit for a growing team
+                </h3>
+                <p>
+                    The Codecov team got started with Sourcegraph quickly and now, they're excited to expand. Already,
+                    support engineers are using Sourcegraph to quickly answer customer questions and the team is looking
+                    to share Sourcegraph with new engineers to help them become code search masters too.
+                </p>
+                <p>
+                    As Codecov, and its codebase, grows, Sourcegraph will be a key way the startup can remain nimble.
+                    “Sourcegraph was an unlock for us,” explained Jarrod. Codecov is growing and ready to further
+                    realize this unlocked potential.
+                </p>
+            </ContentSection>
+
+            <ContentSection color="white" className="pb-7 max-w-700 col-xl-5 text-center">
+                <h5 className="mb-4 text-xl font-weight-bold max-w-650">About Codecov</h5>
+                <p>
+                    Codecov is a small organization making an outsized impact on the lives of over one million
+                    developers. In over 29,000 companies across the globe, Codecov customers are able to ship healthier
+                    code using its code coverage tool. Developers rely on Codecov to provide actionable visibility into
+                    their code coverage across any tech stack.
+                </p>
+            </ContentSection>
```

Comment:
Once the author section is ready, we'll use `<AuthorBio />` here. See [here](https://www.figma.com/file/hUNtMXjxY69aAvsaJOa93S/CodeCov-case-study?node-id=108%3A4569) for reference.

Reply:
Cool, that'll be a nice addition ✨